src/simulator/simulator.cpp

```diff
@@ -119,11 +119,27 @@ void Simulator::init(ros::NodeHandle& n) {
       "/simulator_visualization", 6);
   posMarkerPublisher = n.advertise<visualization_msgs::Marker>(
       "/simulator_visualization", 6);
+  objectLinesPublisher = n.advertise<visualization_msgs::Marker>(
+								 "/simulator_visualization", 6);
```

Comment:
Style Note: Follow the same indent style as other publishers here. Two tabs on next line.